src/request/export/Export.js

```diff
@@ -34,14 +34,46 @@ class Export { // eslint-disable-line no-unused-vars
             this._fileExportSuccessful.bind(this),
             this._reject.bind(this));
 
+        if (this._request.wordsOnly) {
+            /** @type {HTMLElement} */
+            (/** @type {HTMLElement} */ (document.getElementById(ExportWords.Pages.RECOVERY_WORDS_INTRO))
+                .querySelector('.page-header-back-button')).classList.add('display-none');
+        } else {
+            /** @type {HTMLElement} */
+            (/** @type {HTMLElement} */ (document.getElementById(ExportFile.Pages.LOGIN_FILE_INTRO))
+                .querySelector('.page-header-back-button')).classList.add('display-none');
+        }
+
+        /** @type {HTMLDivElement} */
+        this._fileSuccessPage = (document.getElementById(Export.Pages.LOGIN_FILE_SUCCESS));
+
+        /** @type {HTMLButtonElement} */
+        const closeRequestButton = (this._fileSuccessPage.querySelector('button'));
```

Comment:
Also, instead of adding your own click handler for this button, you could make it an `a` and give it the class `.page-header-back-button` and have it handled automatically by the global back button handler.

Reply:
good point. will add the `.page-header-back-button`